src/data/java/dev/socketmods/socketnukes/datagen/utils/advancment/AdvancementProvider.java

```diff
@@ -4,6 +4,7 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import net.minecraft.advancements.Advancement;
+import net.minecraft.data.CachedOutput;
 import net.minecraft.data.DataGenerator;
 import net.minecraft.data.DataProvider;
 import net.minecraft.data.HashCache;
```

Comment:
No need to import `HashCache`, its no longer used, neither is the `GSON` field in the class